docs/understanding-your-pipeline/what-is-snowplow-mini/index.md

```diff
@@ -4,13 +4,27 @@ date: "2020-02-24"
 sidebar_position: 20
 ---
 
-[Snowplow Mini](/docs/pipeline-components-and-applications/snowplow-mini/index.md) is a small, single instance version of Snowplow that primarily serves as a development sandbox which gives you a quick way to debug tracker updates and changes to your schema and pipeline configuration.
+[Snowplow Mini](/docs/pipeline-components-and-applications/snowplow-mini/index.md) is a single-instance version of Snowplow that primarily serves as a development sandbox, giving you a quick way to debug tracker updates and changes to your schema and pipeline configuration.
+
+:::tip
+
+Snowplow Mini is similar to [Snowplow Micro](/docs/getting-started-with-micro/index.md), with the following differences:
+* Micro is more portable and can easily run on your machine or in automated tests.
+* Mini has more features, including enrichments and an OpenSearch Dashboards UI.
+
+:::
 
 You might use Snowplow Mini when:
 
 - You've updated a schema in your Development environment and wish to send some test events against it before promoting it to Production
 - You want to enable an Enrichment in a test environment before enabling it on Production
 
+## Getting started
+
+Snowplow BDP users can request a Snowplow Mini instance through the console (go to “Environments” → “Sandboxes” → “Setup a sandbox”).
```

Comment:
I think this as a code block would look nicer on the page?`“Environments” → “Sandboxes” → “Setup a sandbox”`